test/unit/utils.test.js

```diff
@@ -94,6 +94,28 @@ describe('driver utils', function () {
     });
 
     describe('#wake', function () {
+      context('when the time since last wake is negative', () => {
+        it('does not execute and returns immediately', () => {
+          executor = makeInterruptibleAsyncInterval(fnSpy, {
+            minInterval: 10,
+            interval: 30,
+            clock: () => {
+              return -50;
+            }
+          });
+          // stop the executor
```

Comment:
can we explain more about why it has to be stopped first? I think this step might not be necessary if we initialize the variable instead of leaving it undefined (see my comment on the other file regarding NaN)